docs/tacc_ls5.md

```diff
@@ -0,0 +1,24 @@
+# nf-core/configs: TACC LS5 Configuration
+
+All nf-core pipelines have been successfully configured for use on the Lonestar 5(LS5) HPC cluster at the [Texas Advanced Computing Center (TACC)](https://www.tacc.utexas.edu/).
+
+To use, run the pipeline with `-profile tacc_ls5`. This will download and launch the [`tacc_ls5.config`](../conf/tacc_ls5.config) which has been pre-configured with a setup suitable for the LS5 HPC cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+Before running the pipeline you will need to load Singularity using the environment module system on ls5. You can do this by issuing the commands below:
+
+```bash
+## Singularity environment modules
+module restore
+module load tacc
+```
+
+:warning: Due to the resource limit of users on the login nodes, and not being able to submit jobs from compute nodes you have to call the `NXF_OPTS` in the same line as the nextflow command. All of this even though the ulimit claims 8GB of virtual memory. Setting it in your `.bashrc` causes it to still fail. It may also run out of memory while building the singularity image and you can download it before running. For how to do that see the  [nf-core offline docs](https://nf-co.re/usage/offline)
+
+```bash
+NXF_OPTS='-Xms2G -Xmx2G -Xss228K' nextflow run nf-core/... -profile tacc_ls5
```

Comment:
Can maybe write this as its own `export NXF_OPTS='-Xms2G -Xmx2G -Xss228K'` line instead? Then recommend that people add this to their `.bashrc` file so that they don't have to remember it for every time that they launch Nextflow...

Reply:
I tried that and it still failed for whatever reason, because their login nodes are pretty picky. I'm not even sure if this would work with a real amount of samples this was just the sarek test data. 

The admins have decided they don't like nextflow running on the login nodes at all, so I'm reaching out to the life sciences group to see why they decided they didn't want nextflow used on their clusters, because I think the benefit outweighs the cost of having a footprint on the login node. I'll make a note that nextflow isn't supported in the docs, and leave this for future reference.

Reply:
Yeah, sounds like a bit of a tricky situation. You can always remind the admins that they could choose to allow job submission from compute nodes, then everyone would be happy 😄

Reply:
> allow job submission from compute nodes
I was told using an entire compute node for job submission was a "waste of resources". 

The life sciences group was helpful and said I might be able to do something like:
```
[compute]$ sbatch my_job.slurm
```
instead do:
```
[compute]$ ssh login1 sbatch my_job.slurm
```
Basically if I can get nextflow to put ssh login1 in front of it's sbatch jobs it should work.

Reply:
Can you not allocate a partial node when you use `sbatch`? Just specify a few gigs of memory / few cpus and then share the node with other jobs?

I don't really understand what you mean with the `ssh` thing - I suspect that you'll have problems even if you managed it though, as Nextflow needs to be able to poll the job scheduler to track job statuses. So if `sbatch` doesn't work then the job tracking probably also won't work.

Reply:
You could definitely only request a small amount of resources. I'm not a slurm expert, so I don't know if they can let other jobs use the rest of the node(Though that would make sense right?).

So you can run sbatch from the compute node, it just fails because you can't submit from anything but the login node. And I'm going to guess nextflow uses sinfo under the hood or some other slurm helper, which those work from the compute nodes. 

Basically the idea would be every time nextflow goes to submit a job and calls `sbatch work/aa/asdsd/.command.run` it would call `ssh login1 sbatch ...` instead, and ssh into the login node, and call submit the sbatch job there, and do all the "computing" on the computer node, working out channels, preparing the next sbatch job, polling the queue, outputting docs, etc.

Reply:
Good luck 😃  👍